modules/ossm-about-multi-cluster-topologies.adoc

```diff
@@ -0,0 +1,33 @@
+// This module is used in the following assemblies:
+
+// * install/ossm-multi-cluster-topology.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-multi-cluster-topologies_{context}"]
+= About multi-cluster mesh topologies 
+
+In a multi-cluster mesh, you install and manage a single {istio} mesh across multiple {ocp-product-title} clusters, enabling communication and service discovery between the services. Two factors determine the multi-cluster mesh topology: control plane topology and network topology. There are two options for each topology. Therefore, there are four possible multi-cluster mesh topology configurations.
+
+* Multi-Primary Single Network: Combines the multi-primary control plane topology and the single network network topology models. 
+
+* Multi-Primary Multi-Network: Combines the Combines the multi-primary control plane topology and multi-network network topology models. 
+
+* Primary-Remote Single Network: Combines the primary-remote control plane topology and the single network network topology models.
+
+* Primary-Remote Multi-Network: Combines the primary-remote control plane topology and the multi-network network topology models.
+
+== Control plane topology models
+
+A multi-cluster mesh must use one of the following control plane topologies:
+
+* Multi-Primary: In this configuration, a control plane resides on every cluster. Each control plane observes the API Servers in all of the other clusters for services and endpoints.
+
+* Primary-Remote: In this configuration, the control plane resides only on one cluster, called the primary cluster. No control plane runs on any of the other clusters, called remote clusters. The control plane on the primary cluster discovers services and endpoints and configures the sidecar proxies for the workloads in all clusters. 
+
+== Network topology models
+
+A multi-cluster mesh must use one of the following network topologies:
+
+* Single network: All clusters reside on the same network and there is direct connectivity between the services in all the clusters. There is no need to use gateways for communication between the services across cluster boundaries.
```

Comment:
Seems like all the similar bullets were init-capped, so just for consistency:
```suggestion
* Single Network: All clusters reside on the same network and there is direct connectivity between the services in all the clusters. There is no need to use gateways for communication between the services across cluster boundaries.
```

Reply:
done